yb-voyager/src/utils/logging.go

```diff
@@ -36,6 +37,92 @@ var ErrExit = func(formatString string, args ...interface{}) {
 	atexit.Exit(1)
 }
 
+// OutputLogLevel represents different types of console messages
+type OutputLogLevel int
+
+const (
+	OutputLogLevelSuccess OutputLogLevel = iota
+	OutputLogLevelInfo    OutputLogLevel = iota
+	OutputLogLevelWarning OutputLogLevel = iota
+	OutputLogLevelError   OutputLogLevel = iota
+	OutputLogLevelNormal  OutputLogLevel = iota
+	OutputLogLevelTitle   OutputLogLevel = iota
+)
+
+var (
+	SuccessColor = color.New(color.FgGreen)
+	InfoColor    = color.New(color.FgCyan)
+	WarningColor = color.New(color.FgYellow)
+	ErrorColor   = color.New(color.FgRed)
+	PathColor    = color.New(color.FgYellow, color.Underline)
+	TitleColor   = color.New(color.FgBlue)
+)
```

Comment:
thanks for standardizing this! will be useful for other commands so that voyager can have a constant output style everywhere